### PR TITLE
Add png export feature

### PR DESCRIPTION
**Pull request by mont5piques**

### CODE REVIEW

File: tools/kete.py [hunk 1]
```diff
@@ -699,6 +704,9 @@ def is_index_or_all(string):
                             type=is_index_or_all,
                             const='all', nargs='?', default='all',
                             help='the index of the drawing to fetch or a literal "all"')
+        parser.add_argument('--format', metavar='{svg|png}',
+                            default='svg',
```
Comment (whot) [inline tools/kete.py:708]: please add `choices = ['svg', 'png']` here so argparse takes care of faulty arguments here.

**Review by whot — CHANGES_REQUESTED** [review]
One minor nitpick and - aren't you leaving the now unused `tuhi/svg.py` in place? That needs to be removed if we're not actually using it anymore.

Otherwise looks good, thanks!

File: tools/kete.py (hunk 1 above)
Comment (mont5piques) [reply tools/kete.py:708]: Done

**Comment by mont5piques** [comment]
Changes are done

**Comment by bentiss** [comment]
Thanks

**Pull request merged**